aiohttp/_http_writer.pyx

```diff
@@ -125,8 +126,9 @@ def _serialize_headers(str status_line, headers):
     cdef Writer writer
     cdef object key
     cdef object val
+    cdef char buf[BUF_SIZE]
 
-    _init_writer(&writer)
+    _init_writer(&writer, buf)
```

Comment:
I think this is safe, but need to double check the failure path so we don't have a use-after-free.

Reply:
`PyBytes_FromStringAndSize` makes a copy and `_release_writer` will always be called so this is safe